layouts/shortcodes/python-venv.html

```diff
@@ -0,0 +1,66 @@
+{{/*
```

Comment:
The following text was in all the places where this shortcode is now used, but wasn't carried over to this file. Was that intentional, or do we need to add it at the top of this file?

```html
<p>
    For Python, before we deploy the stack, the following commands need to be run to create a virtual environment, activate it, and install dependencies:
</p>
```

Reply:
Not intentional. I think I just missed it when copy and pasting into the shortcode.